test/__tests__/data/braze_output.json

```diff
@@ -1038,5 +1038,76 @@
   {
     "statusCode": 400,
     "message": "Invalid email, email must be a valid string"
+  },
+  {
+    "body": {
+      "FORM": {},
+      "JSON": {
+        "attributes": [
+          {
+            "_update_existing_only": false,
+            "city": "Disney",
+            "country": "USA",
+            "email": null,
```

Comment:
Email is still coming as null

Reply:
we want to allow email as this is reserved keys and should pass as it is.

Reply:
earlier code if (value || (value === null && reservedKeys.includes(destKey))) { ... is checking against reservedKeys ..even if it is null